meeteval/wer/wer/time_constrained.py

```diff
@@ -93,7 +95,30 @@ def character_based(interval, words):
     ]
 
 
-def character_based_points(interval, words):
+def phoneme_based(interval, words, language):
+    """Divides the interval into one interval per word where the size of the interval is
+    proportional to the number of phonemes in the word."""
+
+    g2p = transphone.read_tokenizer(language)
```

Comment:
The call transphone.read_tokenizer sounds, that it loads a model. Has it a caching? 
If not, we should do a caching, since this function is called for every segment.